docs/modeling/physical_models.rst

```diff
@@ -66,9 +66,14 @@ default units returned by :class:`~astropy.modeling.physical_models.BlackBody`.
     plt.tight_layout()
     plt.show()
 
-The ``bolometric_flux()`` member function gives the bolometric flux using
+The ``bolometric_flux`` member function gives the bolometric flux using
```

Comment:
Why not use Sphinx API linking like this?
```
:meth:`~astropy.modeling.physical_models.BlackBody.bolometric_flux`
```
Same comment applies to other functions with API doc that is not linked.

Reply:
Will do.